homework7/contacts.php

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```

Comment:
@Niki1993 
It will look better if this page follows the template you have created for the other two : )